client/src/util/helpers.ts

```diff
@@ -109,6 +109,11 @@ export default {
     return null;
   },
 
+  formatPrice(locale: string, amount: number) {
+    const formatPrice = new Intl.NumberFormat(locale);
+    return formatPrice.format(amount);
+  },
+
```

Comment:
Two things here:
1. It might be good to make locale an optional variable (by adding a ? at the end of the name) and including the default locale string within the body of the function, in the same way that `formatDate` works.
2. It would be good to specify the locale variable as a `SupportedLocale` here, as that will allow us to type check and make sure we are inputting a locale that we actually have enabled on the site, and not just any random string. Again, `formatDate` is a good example of that.